assets/stylesheets/kitten/helpers/typography/_font-scale.scss

```diff
@@ -0,0 +1,43 @@
+/// Apply a font scale using according to media view.
+///
+/// Font-step will be computed directly using the modular scale. It is optional
+/// and set to 0 by default the regular font step.
+///
+/// Font-step-on-media is optional. If it is not defined, it will be set to the
+/// regular font step.
+///
+/// media-size will be computed directly using the modular scale. It is
+/// optional and is set to media 's' by default. It's the minimum media size
+/// from where the large font step has to be applied.
```

Comment:
Après réflexion, je me dit qu'on pourrait avoir une petite font si on le voulait sur le breakpoint choisi. Est-ce que ça te semble correct si on met `from where a specified font step (font-step-on-media) has to be applied.` ?